### PR TITLE
changed channel display style and removed time stamp

### PR DESCRIPTION
**Pull request by Ritika8081**
@akadeepesh channel will display in grid form, for small devices it will be in list view.
time stamps are removed .

### CODE REVIEW

**Comment by akadeepesh** [comment]
@Ritika8081 It will just take a useState to provide user a option to change the display and orientation of charts, please discuss with Deepak sir if we should do that. 

Also I was thinking about creating a power stop button on footer, which will pause all charts on one click, because in real life situation if someone wants to pause the stream at a specific time, most probably he/she want to pause stream of all channel at that particular time.

File: src/components/Canvas.tsx [hunk 1]
```diff
@@ -296,31 +296,33 @@ const Canvas: React.FC<CanvasProps> = ({ data, selectedBits }) => {
   };
 
   return (
-    <div className="flex justify-center items-center flex-row md:h-[85%] h-[80%] w-screen px-4 gap-10">
-      <div className="flex justify-center items-center flex-col h-[85%] w-[95%]">
+    <div className="flex justify-center items-start md:h-[85%] h-[80%] w-screen p-2">
```
Comment (akadeepesh) [inline src/components/Canvas.tsx:299]: items-start is a default property we can remove that.

**Review by akadeepesh — APPROVED** [review]

File: src/components/Canvas.tsx (hunk 1 above)
Comment (Ritika8081) [reply src/components/Canvas.tsx:299]: If I am removing ( items-start ), design is getting affected it is not coming as default.

**Comment by Ritika8081** [comment]
Yes we can provide user a option to change the display and orientation of charts.
we can  create a power stop button on footer.

**Pull request merged**